common-authentication/src/main/java/org/alfresco/hxi_connector/common/adapters/auth/AuthService.java

```diff
@@ -51,35 +51,43 @@ public void setAlfrescoAuthorizationHeaders(Exchange exchange)
     {
         AuthProperties.AuthProvider authProvider = authProperties.getProviders().get(ALFRESCO_AUTH_PROVIDER);
         String authType = authProvider.getType();
-        String authHeaderValue = getAlfrescoAuthHeaderValue(accessTokenProvider, authProvider);
         clearAuthHeaders(exchange);
-        exchange.getIn().setHeader(AUTHORIZATION, authHeaderValue);
+        exchange.getIn().setHeader(AUTHORIZATION, getAlfrescoAuthHeaderValue(authProvider));
         log.debug("Authorization :: {} {} authorization header added", ALFRESCO_AUTH_PROVIDER, authType);
     }
 
     public void setHxIAuthorizationHeaders(Exchange exchange)
     {
-        final String token = accessTokenProvider.getAccessToken(HXI_AUTH_PROVIDER);
         AuthProperties.AuthProvider authProvider = authProperties.getProviders().get(HXI_AUTH_PROVIDER);
-        String authHeaderValue = BEARER + token;
         clearAuthHeaders(exchange);
-        exchange.getIn().setHeader(AUTHORIZATION, authHeaderValue);
+        exchange.getIn().setHeader(AUTHORIZATION, getBearerAuthHeader(HXI_AUTH_PROVIDER));
         exchange.getIn().setHeader(ENVIRONMENT_KEY_HEADER, authProvider.getEnvironmentKey());
         log.debug("Authorization :: {} authorization header added", HXI_AUTH_PROVIDER);
     }
 
-    private String getAlfrescoAuthHeaderValue(AccessTokenProvider accessTokenProvider, AuthProperties.AuthProvider authProvider)
+    public String getAuthHeaderValue(String providerId)
+    {
+        AuthProperties.AuthProvider authProvider = authProperties.getProviders().get(ALFRESCO_AUTH_PROVIDER);
+        return providerId.equals(HXI_AUTH_PROVIDER) ? getBearerAuthHeader(HXI_AUTH_PROVIDER) : getAlfrescoAuthHeaderValue(authProvider);
+    }
+
+    private String getAlfrescoAuthHeaderValue(AuthProperties.AuthProvider authProvider)
```

Comment:
This `Value` suffix is required here? Other method: `getBearerAuthHeader()` doesn't have it in it's name 🤔

Reply:
I don't think I changed the name of that method with this PR, but it's a good point.
On the other hand, none of these `get*Header*` methods is returns headers per se but only their values.
I am happy to rename all of them to be more consistent (and remove `value` part from all of them if that's ok).

Reply:
Fine with me too :+1: